### DISCUSSION TITLE
Pulse detection through S0 instead of LED detection

### DISCUSSION THREAD

**Discussion by tomasenskede** [klaasnicolaas/home-assistant-glow#590] [Q&A]
I have access to the S0-port, so I don't need to rely on reading the LED blinks. Can I still use the ESP32 with this setup?

**Comment by huizebruin** [comment]
Hello @tomasenskede .

Or you can look at the https://s0tool.nl 
That one has a port for counting the s0 port of your counter.

I will update the website for pictures how you can make it yourself or buy it if you live in the Netherlands or Belgium. 

Or my github. 
https://github.com/huizebruin/s0tool

**Reply by klaasnicolaas** [reply]
For the record, I asked @huizebruin to comment on this because he made an awesome project that works with the S0 port 😍 

./Klaas
